src/librustdoc/html/static/main.js

```diff
@@ -1167,6 +1143,22 @@ function hideThemeButtonState() {
         var hideMethodDocs = getSettingValue("auto-hide-method-docs") === "true";
         var hideImplementors = getSettingValue("auto-collapse-implementors") !== "false";
         var hideLargeItemContents = getSettingValue("auto-hide-large-items") !== "false";
+        var hideTraitImplementations =
+            getSettingValue("auto-hide-trait-implementations") !== "false";
+
+        var impl_list = document.getElementById("trait-implementations-list");
+        if (impl_list !== null) {
+            onEachLazy(impl_list.getElementsByClassName("collapse-toggle"), function(e) {
+                collapseNonInherent(e, collapse);
+            });
+        }
+
+        var blanket_list = document.getElementById("blanket-implementations-list");
+        if (blanket_list !== null) {
+            onEachLazy(blanket_list.getElementsByClassName("collapse-toggle"), function(e) {
+                collapseNonInherent(e, collapse);
```

Comment:
Where is this `collapse` coming from?

Reply:
Thanks for spotting. This was a leftover param that was only true when `getSettingValue("collapse") === "true"`. That was never the case, because `"collapse"` was the wrong setting name anyhow ("rustdoc-collapse" was the right setting name). Removed it.